### ISSUE TITLE
Watson Text-to-speach stopped working

### ISSUE THREAD

**Issue by woj-tek** [lovac42/awesometts-CCBC-addon#2]
Recently (since a couple of days ago) attempts to use Watson service results in:

> Cannot record the input phrase with these settings.
> 'NoneType' object has no attribute 'replace'

**Comment by lovac42** [comment]
Yeah I noticed, Bing as well. I'll find time to look into it, but these types of situations are going to happen a lot sadly.

**Comment by woj-tek** [comment]
I'm fully aware that they may and will happen - services do love changing their APIs. Just though I'll report it here "for the record" :-)

**Comment by lovac42** [comment]
It looks like they removed the demo site. Not sure if it's permanent, so we'll have to wait and see. Except for Bing, I fixed up the other ones and did a new release. https://github.com/lovac42/awesometts-CCBC-addon/releases

Keep this open as a reminder so we could check up on it later.

**Comment by woj-tek** [comment]
Thanks for the update!

**Comment by lovac42** [comment]
Seems the service is back online. So they took it offline to update the site from time to time.

**Comment by woj-tek** [comment]
Aaand it's gone again with:
```
Cannot preview the input phrase with these settings.

No response for web request
```

Demo website seems to [still be there](https://text-to-speech-demo.ng.bluemix.net/) though, looking at the [pricing](https://www.ibm.com/cloud/watson-text-to-speech/pricing) maybe it could/would be a good idea to have it as an option? 10k characters monthly seems quite hefty

**Comment by lovac42** [comment]
Fixed: https://github.com/lovac42/awesometts-CCBC-addon/releases/download/v1.18.2/awesometts_v1.18.2_Anki21.ankiaddon

I don't want to sign up for anything, as well as read though their API documentation & requirements. But I'll consider it if it keeps happening.

**Comment by woj-tek** [comment]
Thank you, it works like a charm!
